bin/lang-js/BUCK

```diff
@@ -24,6 +24,8 @@ alias(
     actual = ":bin"
 )
 
+# we include si-gen-cloud-control here because the deno workspace
+# requires resolution of all members even if we don't use it
```

Comment:
Yeah that's a bummer, but cool.

I bet we might be able to figure something out, similar to using `turbo`'s trick of simulating our mono repo with the appropriate directories and files to make dependency installs work.

Reply:
That's a great idea. I want to take another pass over the deno jobs in general, so I'll take a look at this then